README.md

```diff
@@ -4,14 +4,44 @@
 
 [github-actions-badge]: https://github.com/volodeyka/event-struct/workflows/CI/badge.svg
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras at nisl mattis,
-convallis velit sed, accumsan leo. Fusce sed dui est. Nullam suscipit tincidunt
-ipsum id accumsan. Sed eget sapien ornare magna fermentum blandit. Maecenas
-commodo porttitor leo, at pellentesque nulla imperdiet vel. Ut iaculis, lectus
-eget sodales bibendum, justo nulla posuere augue, eu rutrum turpis dui ac erat.
-Donec pharetra elit vitae tincidunt vestibulum. Sed iaculis facilisis ipsum sed
-aliquam. Nullam tincidunt purus sed est pulvinar lacinia. Integer at nunc vitae
-sem pulvinar maximus quis in ipsum. Duis ut sem varius, ultrices leo vel,
-interdum orci. Etiam bibendum varius lectus ac scelerisque. Nam in magna sapien.
-Nunc mattis orci non risus rutrum posuere. Mauris nec viverra lectus.
+A library of formalized event structure theory applied to weak-memory models.
+Includes notion and theory of execution event structures, labeled transition
+system on execution event structures, big-step semantics for simple
+register machine, and well-founded relations theory.
 
+## Files in poject
```

Comment:
It is a good idea to give a quick description of each file  :+1: 

I would only change the header to `Description of Files`